src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -246,6 +246,65 @@ public void GetFolderPath_OSX_NonEmptyFolderPaths(Environment.SpecialFolder fold
             }
         }
 
+        [Theory]
```

Comment:
This test needs to be in a separate file that is only included in netstandard1.7 build of test assembly.  That is the cause of your CI failures.